packages/table/src/table.tsx

```diff
@@ -806,10 +849,6 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
      * intend to redraw the region layer.
      */
     private maybeRenderRegions(getRegionStyle: IRegionStyler) {
-        if (this.isGuidesShowing()) {
-            return undefined;
-        }
```

Comment:
K, so I remember now. With this check in place, selected regions don't appear while you're reordering (b/c reordering shows a guide, causing this function to return before drawing the selected region). And then selected regions remain broken even after the reordering completes for some reason.

So will need to remove after all!